i18n/en.i18n.json

```diff
@@ -79,6 +79,9 @@
   "remove_owner_from_hangout": "Hey! You can't remove yourself from your hangout.",
   "delete_hangout_confirm": "Are you sure?",
   "delete_hangout_text": "Note: All participants will be notified via email of cancellation",
+  "continue_popup_text":"By continuing, I agree that I am at least 13 years old and have read and agree to the terms of service and privacy policy.",
```

Comment:
@sergeant-q Updated PR. Check it out and feel free to let me know if there is anything else I missed.